### PR TITLE
https://issues.redhat.com/browse/ACM-3370 Placement rewrite

### PR DESCRIPTION
**Pull request by oafischer**
Needs technical review and writing review. This is a restructure and rewrite to comply with our current doc standards.

### CODE REVIEW

File: clusters/cluster_lifecycle/placement_filter.adoc [hunk 1]
```diff
@@ -0,0 +1,329 @@
+[#placement-labelselector-claimSelector]
+== Filtering _ManagedClusters_ from _ManagedCusterSets_
+
+You can select which `ManagedClusters` to filter by using `labelSelector` or `claimSelector` . See the following examples to learn how to use both filers:
```
Comment (cadawson) [inline clusters/cluster_lifecycle/placement_filter.adoc]:
```suggestion
You can select which `ManagedClusters` to filter by using `labelSelector` or `claimSelector`. See the following examples to learn how to use both filers:
```

**Review by cadawson — COMMENTED** [review]
One minor typo. Thanks for working on that.

File: clusters/cluster_lifecycle/placement_decision.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
+[#placement-decision]
+== Filtering _ManagedClusters_ by using _PlacementDecisions_
```
Comment (haoqing0110) [inline clusters/cluster_lifecycle/placement_decision.adoc]: Checking selected ManagedClusters by using PlacementDecisions.

File: clusters/cluster_lifecycle/placement_intro.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[#placement-intro]
+= Filtering _ManagedClusters_ with placement
```
Comment (haoqing0110) [inline clusters/cluster_lifecycle/placement_intro.adoc]: Selecting _ManagedClusters_ with placement. 
Placement will actually filter and prioritize clusters, and select the top N cluster. So avoid using Filtering in some general title.

File: clusters/cluster_lifecycle/placement_filter.adoc [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+    - requiredClusterSelector:
+        claimSelector:
+          matchExpressions:
+            - key: region.open-cluster-management.io
+              operator: In
+              values:
+                - us-west-1
+----
+
+You can also select how many `ManagedClusters` you want to filer by using the `numberOfClusters` paremeter. See the following example:
```
Comment (haoqing0110) [inline clusters/cluster_lifecycle/placement_filter.adoc]: filter

File: clusters/cluster_lifecycle/placement_filter.adoc [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+  tolerations:
+    - key: cluster.open-cluster-management.io/unreachable
+      operator: Exists
+      tolerationSeconds: 300 <1>
+----
++
+<1> Specify after how many seconds you want the workload to be transferred.
+
+[#placement-prioritizerpolicy]
+== Filtering _ManagedClusters_ by defining _prioritizerPolicy_ with placement
```
Comment (haoqing0110) [inline clusters/cluster_lifecycle/placement_filter.adoc]: Prioritize _ManagedClusters_ by defining _prioritizerPolicy_ with placement

File: clusters/cluster_lifecycle/placement_filter.adoc [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+
+View the following examples to learn how to filter `ManagedClusters` by using the `prioritizerPolicy` paremeter with placement.
```
Comment (haoqing0110) [inline clusters/cluster_lifecycle/placement_filter.adoc]: learn how to prioritize

**Comment by oafischer** [comment]
Thanks for the feedback @haoqing0110! I implemented the changes, let me know if it looks good now.

File: clusters/cluster_lifecycle/cluster_lifecycle_intro.adoc [hunk 7]
```diff
@@ -42,7 +42,10 @@ See the following documentation:
 ** xref:../cluster_lifecycle/create_clustersetbinding.adoc#creating-managedclustersetbinding[Creating a _ManagedClusterSetBinding_ resource]
 ** xref:../cluster_lifecycle/taints_tolerations.adoc#taints-tolerations-managed[Placing managed clusters by using taints and tolerations]
 ** xref:../cluster_lifecycle/remove_cl_clusterset.adoc#removing-cluster-managedclusterset[Removing a managed cluster from a _ManagedClusterSet_]
-* xref:../cluster_lifecycle/placement_managed.adoc#placement-managed[Using ManagedClusterSets with Placement]
+* xref:../cluster_lifecycle/placement_intro.adoc#placement-intro[Selecting _ManagedClusters_ with placement]
+** xref:../cluster_lifecycle/placement_overview.adoc#placement-overview[Placement overview]
+** xref:../cluster_lifecycle/placement_filter.adoc#placement-labelselector-claimSelector[Filtering _ManagedClusters_ from _ManagedCusterSets_]
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cluster_lifecycle_intro.adoc]:
```suggestion
** xref:../cluster_lifecycle/placement_filter.adoc#placement-labelselector-claimSelector[Filtering _ManagedClusters_ from _ManagedClusterSets_]
```

File: clusters/cluster_lifecycle/placement_decision.adoc [hunk 8]
```diff
@@ -0,0 +1,41 @@
+[#placement-decision]
+= Checking selected _ManagedClusters_ by using _PlacementDecisions_
+
+One or more `PlacementDecision` kinds with the label `cluster.open-cluster-management.io/placement={placement_name}` are created to represent `ManagedClusters` selected by a placement.
+
+If a `ManagedCluster` is selected and added to a `PlacementDecision`, components that consume this placement might apply the workload on this `ManagedCluster`. After the `ManagedCluster` is no longer selected and is removed from the `PlacementDecision`, the workload that is applied on this `ManagedCluster` is removed.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/placement_decision.adoc]:
```suggestion
If a `ManagedCluster` is selected and added to a `PlacementDecision`, components that consume this placement might apply the workload on this `ManagedCluster`. After the `ManagedCluster` is no longer selected and is removed from the `PlacementDecision`, the workload that is applied on this `ManagedCluster` is removed. See _PlacementDecisions API_ to learn more about the API.
```

File: clusters/cluster_lifecycle/placement_decision.adoc [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+See _PlacementDecisions API_ to learn more about the API.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/placement_decision.adoc]:
```suggestion

```

File: clusters/cluster_lifecycle/placement_filter.adoc [hunk 10]
```diff
@@ -0,0 +1,329 @@
...
+        labelSelector:
+          matchExpressions:
+            - key: feature.open-cluster-management.io/addon-application-manager
+              operator: DoesNotExist
+----
+
+[#placement-filter-resources]
+== Additional resources
+
+- See https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable[Node Allocatable] for more details.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/placement_filter.adoc]:
```suggestion
- See link:https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable[Node Allocatable] for more details.
- Return to xref:../clusters/cluster_lifecycle/placement_intro.adoc#placement-intro[Selecting ManagedClusters with placement] for other topics.
```

File: clusters/cluster_lifecycle/placement_overview.adoc [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+*Notes:* 
+
+* You must bind at least one `ManagedClusterSet` to a namespace by creating a `ManagedClusterSetBinding` in that namespace. 
+* You must have role-based access to `CREATE` on the virtual sub-resource of `managedclustersets/bind`.
+
+[#placement-resources]
+== Additional resources
+
+- See xref:../cluster_lifecycle/taints_tolerations.adoc#taints-tolerations-managed[Using taints and tolerations to place managed clusters] for more information.
+- See xref:../api/placement.json.adoc#placements-clusters-api[Placements API] to learn more about the API.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/placement_overview.adoc:29]:
```suggestion
- See xref:../api/placement.json.adoc#placements-clusters-api[Placements API] to learn more about the API.
- Return to xref:../clusters/cluster_lifecycle/placement_intro.adoc#placement-intro[Selecting _ManagedClusters_ with placement].
```

**Review by dockerymick — APPROVED** [review]
Updates look good! I made a few suggestions, otherwise this looks great!

File: clusters/cluster_lifecycle/placement_filter.adoc [hunk 12]
```diff
@@ -0,0 +1,329 @@
+[#placement-labelselector-claimSelector]
+= Filtering _ManagedClusters_ from _ManagedCusterSets_
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/placement_filter.adoc]:
```suggestion
= Filtering _ManagedClusters_ from _ManagedClusterSets_
```

Comment (oafischer) [reply clusters/cluster_lifecycle/placement_filter.adoc]: Great catch, needed a moment to see the difference even with the correction right in front of me lol

**Pull request merged**